src/analog/commands/uplink/getChannelsStatus.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * Information about the current status of channels on the device,
+ *
+ * @packageDocumentation
+ *
+ * @example create command instance from command body hex dump
+ * ```js
+ * import * as getChannelsStatus from 'jooby-codec/analog/commands/uplink/getChannelsStatus.js';
+ *
+ * const bytes = [0x1f, 0x32, 0x03, 0x02, 0x00, 0x01];
+ *
+ * // decoded payload
+ * const parameters = getChannelsStatus.fromBytes(bytes);
+ *
+ * console.log(parameters);
+ * // output:
+ * [{
+ *     type: 2,
+ *     status: {
+ *         channel: 0,
+ *         state: true
+ *     }
+ * }]
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/analog/commands/GetChannelsStatus.md#response)
+ */
+
+import BinaryBuffer, {IBinaryBuffer} from '../../../utils/BinaryBuffer.js';
+import * as types from '../../../types.js';
+import * as command from '../../utils/command.js';
+import * as channelsTypes from '../../constants/channelsTypes.js';
+
+
+export const id: types.TCommandId = 0x321f;
+export const name: types.TCommandName = 'getChannelsStatus';
+export const headerSize = 3;
+
+
+interface IBinarySensorStatus {
```

Comment:
it's better to put interfaces before constants

Reply:
fixed